akka-docs/src/main/paradox/scala/persistence.md

```diff
@@ -638,7 +638,7 @@ akka.persistence.journal.leveldb.replay-filter {
 <a id="snapshots"></a>
 ## Snapshots
 
-Snapshots can dramatically reduce recovery times of persistent actors and views. The following discusses snapshots
+Some actors in your app may be bound to have extremely long event logs. In that case,  you can use snapshots to reduce recovery times drastically. The following discusses snapshots
```

Comment:
I like giving some more context here. Perhaps even suggest that you might also consider splitting out into many shorter-lived actors?